ibm/service/cos/resource_ibm_cos_bucket_lifecycle_configuration.go

```diff
@@ -105,14 +105,80 @@ func ResourceIBMCOSBucketLifecycleConfiguration() *schema.Resource {
 							Type:             schema.TypeList,
 							Required:         true,
 							DiffSuppressFunc: suppressMissingFilterConfigurationBlock,
-							// IBM has filter as required parameter
+							// IBM has filter a required parameter
 							MaxItems: 1,
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
 									"prefix": {
 										Type:     schema.TypeString,
 										Optional: true,
 									},
+									"object_size_greater_than": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										ValidateFunc: validation.IntAtLeast(0),
+									},
+									"object_size_less_than": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										ValidateFunc: validation.IntAtLeast(1),
+									},
+									"tag": {
+										Type:     schema.TypeList,
+										MaxItems: 1,
+										Optional: true,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"key": {
+													Type:     schema.TypeString,
+													Required: true,
+												},
+												"value": {
+													Type:     schema.TypeString,
+													Required: true,
+												},
+											},
+										},
+									},
+									"and": {
```

Comment:
Can user provide combination of multiple filters along with other filter (tags, prefix...so on) if not can we add some conflicts with can't provider filter.0.and with other arguments

Reply:
We cannot add other filters with each other.
But the parent element lifecycle_rule can multiple rules so we cant add the conflict like lifecycle.0.filter. to the arguments.